sdk/tables/aztables/recordings/TestServiceClient_Cosmos/TestCreateTable.yaml

```diff
@@ -0,0 +1,75 @@
+---
+version: 1
+interactions:
+- request:
+    body: '{"TableName":"gotablecwmvbx3xi23m7"}'
+    form: {}
+    headers:
+      Accept:
+      - application/json;odata=minimalmetadata
+      Authorization:
+      - sanitized
+      Content-Length:
+      - "36"
+      Content-Type:
+      - application/json
+      Dataserviceversion:
+      - "3.0"
+      User-Agent:
+      - azsdk-go-tables/<version> azsdk-go-tables/<version> azcore/v0.13.4 (go1.16.2;
```

Comment:
The telemetry string in the generated code needs to be updated to replace `<version>` with `v0.1.0`.
Also just noticed shouldn't it have generated with `azsdk-go-aztables`? instead of just ending with `tables`